packages/web-react/src/types/shared/dictionaries.ts

```diff
@@ -9,7 +9,6 @@ import {
   ValidationStates,
 } from '../../constants';
 
-/* Alignment */
```

Comment:
We probably want to keep that (but it should actually be `Alignment X`)

Reply:
It could be also AlignmentY. Below are similar colors with a uniform caption.